src/tactic/slim_check.lean

```diff
@@ -103,6 +105,9 @@ open tactic slim_check
 declare_trace slim_check.instance
 declare_trace slim_check.decoration
 declare_trace slim_check.discared
```

Comment:
Here and elsewhere in this file: is `discared` a typo for `discarded`?

Reply:
Indeed!